presto-cli/src/main/java/io/prestosql/cli/ClientOptions.java

```diff
@@ -406,7 +407,7 @@ public ClientExtraCredential(String extraCredential)
             checkArgument(nameValue.size() == 2, "Extra credential: %s", extraCredential);
 
             this.name = nameValue.get(0);
-            this.value = nameValue.get(1);
+            this.value = urlDecode(nameValue.get(1));
```

Comment:
user should not need to url-encode credentials on command line. 

IMO we should just remove ` checkArgument(nameValue.size() == 2` check above

Reply:
BTW can you add a test for this? :)